assets/css/colors.js

```diff
@@ -1,4 +1,7 @@
 const colors = {
+  core: {
+    'off-white': '#F2F2F2',
```

Comment:
Junjie is using the `core` collection of colours in Figma for the designs.
I will add the set of tokens in a separate PR.

Slack thread: https://centrapay.slack.com/archives/CTM2483GW/p1671564336450279 🧵